cfme/tests/integration/test_cfme_auth.py

```diff
@@ -420,3 +420,73 @@ def _cleanup():
         for group in retrieved_groups:
             if group.exists:
                 group.delete()
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.customer_scenario
+@pytest.mark.meta(coverage=[1710998])
+def test_verify_datastores_list():
+    """
+    Datastore filter should work properly after even delete the datastores
+    Bugzilla:
+        1710998
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Auth
```

Comment:
These manual test cases shouldn't be added to this module, as the module is marked to use the auth fixtures on all test functions. This is resulting in an error condition when the tests are collected:

```
INTERNALERROR>   File "/cfme/cfme_tests/cfme/markers/uncollect.py", line 115, in uncollectif
INTERNALERROR>     raise Exception(f'uncollectif {func_name} hit KeyError, '
INTERNALERROR> Exception: uncollectif test_verify_datastores_list[ldap-ad_bos-cn-test user1] hit KeyError, best guess a fixture was not ready
```

These tests are more related to RBAC than they are related to authentication - they're not really about testing authentication to CFME, but about testing authorization for access to specific pages or application of rules across roles.

Reply:
@mshriver thank you for review.